warp/tests/test_paddle.py

```diff
@@ -8,10 +8,35 @@
 import unittest
 
 import numpy as np
+import paddle
```

Comment:
Hey @HydrogenSulfate, this top-level import also causes issues if `paddle` is not installed.